docker/elasticsearch/default.env

```diff
@@ -1,3 +0,0 @@
-ES_JAVA_OPTS=-Xms512m -Xmx512m
-bootstrap.memory_lock=true
```

Comment:
`bootstrap.memory_lock=true`

The above parameter prevents ES (at the process level) from using swap. It is independent of OS and rather recommended.

[https://www.elastic.co/guide/en/elasticsearch/reference/master/setup-configuration-memory.html](https://www.elastic.co/guide/en/elasticsearch/reference/master/setup-configuration-memory.html)

Reply:
This setting must be considered for a production environment. By default, Elasticsearch assumes that we are working on a development environment as per https://www.elastic.co/guide/en/elasticsearch/reference/current/system-config.html.